t/alien.t

```diff
@@ -49,6 +62,9 @@ xs_ok $xs, with_subtest {
 END {
     system( qw[ xpaset -p xpamb -exit ] )
       unless $xpamb_already_running;
+    if ($win32_processobj) { #on Windows `xpaset -p xpamb -exit` currently does not work
+      $win32_processobj->Kill(0);
+    }
 }
 
```

Comment:
You're correct, there really should be a guaranteed kill of the `xpamb` process. The fact that it doesn't work (as indicated by the test log) indicates there's a bigger probelm at foot.